corehq/apps/domain/views.py

```diff
@@ -1644,7 +1644,7 @@ class ConfirmSubscriptionRenewalView(DomainAccountingSettings, AsyncHandlerMixin
     @property
     @memoized
     def next_plan_version(self):
-        new_edition = self.request.POST.get('plan_edition').title()
+        new_edition = self.request.POST.get('plan_edition', "").title()
```

Comment:
It happens when you do a GET on the url that the POST is failing on. I think it's just a consequence of the user trying to refresh that page after it fails. Maybe we should disallow GETs on that view though?